pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -56,61 +181,120 @@ class LightningTensor(Device):
+    recommended.
+
+    Currently, only the Matrix Product State (MPS) method as implemented in the ``cutensornet`` backend is supported.
+
     Args:
         wires (int): The number of wires to initialize the device with.
             Defaults to ``None`` if not specified.
+        max_bond_dim (int): The maximum bond dimension to be used in the MPS simulation. Default is 128.
+            The accuracy of the wavefunction representation comes with a memory tradeoff which can be
+            tuned with `max_bond_dim`. The larger the internal bond dimension, the more entanglement can
+            be described but the larger the memory requirements. Note that GPUs are ill-suited (i.e. less
+            competitive compared with CPUs) for simulating circuits with low bond dimensions and/or circuit
+            layers with a single or few gates because the arithmetic intensity is lower.
+        cutoff (float): The threshold used to truncate the singular values of the MPS tensors. Default is 0
+        cutoff_mode (str): Singular value truncation mode. Options: ["rel", "abs"].
         backend (str): Supported backend. Currently, only ``cutensornet`` is supported.
         method (str): Supported method. Currently, only ``mps`` is supported.
         shots (int): How many times the circuit should be evaluated (or sampled) to estimate
             the expectation values. Currently, it can only be ``None``, so that computation of
             statistics like expectation values and variances is performed analytically.
         c_dtype: Datatypes for the tensor representation. Must be one of
             ``np.complex64`` or ``np.complex128``.
-        **kwargs: keyword arguments. TODO add when cuTensorNet MPS backend is available as a prototype.
+        **kwargs: keyword arguments.
+
+    **Example**
+
+    .. code-block:: python
+
+        import pennylane as qml
+
+        num_qubits = 100
+
+        dev = qml.device("lightning.tensor", wires=num_qubits)
+
+        @qml.qnode(dev)
+        def circuit(num_qubits):
+            for qubit in range(0, num_qubits - 1):
+                qml.CZ(wires=[qubit, qubit + 1])
+                qml.X(wires=[qubit])
+                qml.Z(wires=[qubit + 1])
+            return qml.expval(qml.Z(0))
+
+    >>> print(circuit(num_qubits))
+    -1.0
     """
 
     # pylint: disable=too-many-instance-attributes
 
     # So far we just consider the options for MPS simulator
-    _device_options = ("backend", "c_dtype")
-
+    _device_options = ("backend", "c_dtype", "max_bond_dim", "cutoff", "cutoff_mode", "method")
+    _CPP_BINARY_AVAILABLE = LT_CPP_BINARY_AVAILABLE
     _new_API = True
 
     # pylint: disable=too-many-arguments
     def __init__(
         self,
         *,
         wires=None,
-        backend="cutensornet",
-        method="mps",
+        max_bond_dim: int = 128,
+        cutoff: float = 0,
+        cutoff_mode: str = "abs",
+        backend: str = "cutensornet",
+        method: str = "mps",
```

Comment:
Please add docstrings for all these default values.

Reply:
Done!